src/shareManagement.js

```diff
@@ -74,6 +73,9 @@ class Shares {
       if (optionalParams.attributes) {
         postData.attributes = optionalParams.attributes
       }
+      if (optionalParams.quicklink) {
+        postData.quicklink = optionalParams.quicklink
```

Comment:
isn't the quicklink flag supposed to go into the `attributes` (see above)? Otherwise we couldn't have it for oc10, because an additional `quicklink` prop wouldn't be handled by the backend.

Reply:
we've decided to keep it on the top level instead of putting it into the attributes (cs3).
For oc10 we decided to catch the quicklink param here and mirrow it into the attributes.

Reply:
how do you decide which one to do (cs3 / oc10)? also, for a migration from oc10 to ocis this means that we need to keep this in mind particularly, instead of just blindly copying all attributes of a link.

Reply:
i do not differentiate the backends, just send both... and add a comment why thats the case... we talked about it this morning and we all agreed `attributes` is more or less like a param blackbox. Having it top-level is easier to document and even more clean.

Are you against it?

Reply:
sending both is fine then. wanted to make sure that we don't have oc10/ocis if/else constructs in the sdk code.

this still has the drawback of having to write the toplevel quicklink flag during migration from oc10 to ocis. please make sure to add that to some migration considerations collection and then go ahead like you decided. 👍

Reply:
will do, thanks for your input. Ill jump back into this pr right after we have merged the web groundwork pr.